lib/Button/index.js

```diff
@@ -55,13 +55,15 @@ class Button extends Component {
   }
 
   render() {
-    const { disabled, children, className } = this.props;
+    const { disabled, children, className, isSubmit } = this.props;
+    const type = (isSubmit) ? 'submit' : 'button';
```

Comment:
When using buttons in forms that use `onSubmit` we get issues where all buttons submit the form. Adding this in resolves that issue.